### PR TITLE
modified text-alignment

### PR DESCRIPTION
**Pull request by sidharthviz**
Screenshot 2023-01-29 at 1 51 56 AM
#169 
feat: Modified the Design 
Seems better now i think.

### CODE REVIEW

**Comment by Wabri** [comment]
Maybe on mobile is a little broken, could you fix it?

**Comment by sidharthviz** [comment]
> 
> 
> Maybe on mobile is a little broken, could you fix it?

Sure, I'll work on it

**Review by sidharthviz — COMMENTED** [review]
Mobile view Fixed... Let me know if needs anymore changes

**Comment by TheJoin95** [comment]
I commented your pr

**Comment by TheJoin95** [comment]
I commented your pr

**Comment by sidharthviz** [comment]
> I commented your pr with the changes needed

Yes, Please tell me what more changes it needs, I've already fix the mobile view.
Screenshot 2023-01-29 at 1 57 11 PM

File: styles/globals.css [hunk 1]
```diff
@@ -368,6 +368,10 @@ div.image .img {
 }
 
 .sponsors_tier {
+  text-align: center;
+  padding: 2rem;
+  margin: 6rem;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```
[the remark this answers is not in the document]
Comment (sidharthviz) [reply styles/globals.css:374]: Oh, Sure i will fix it with the previous one.

**Review by sidharthviz — COMMENTED** [review]
#173  Fixed font

**Comment by 404answernotfound** [comment]
I liked the left alignment more as the entire page is left aligned so centering text doesnt really fit. PR is good but I'm not sure we should merge.

**Comment by TheJoin95** [comment]
I agree with @404answernotfound

**Comment by 404answernotfound** [comment]
Thank you for your help @sidharthviz 

We will be closing this PR based upon public vote. There are other issues that you might be interested in taking so let us know if we can help you with any of those :)

**Pull request closed without merging**